.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,114 @@
+name: Node.js CI
+
+on: [push, pull_request]
```

Comment:
Do we need the tests to run on pushes prior to a pull request being made? (This is to avoid running tests twice unnecessarily)

Reply:
I'm not too bothered by this. I think it's good to know if a commit passes CI even if it doesn't exist within a pull request.